README.Rmd

```diff
@@ -23,14 +23,57 @@ A shiny app for automated detection of signals in infectious disease surveillanc
 This app allows the user to investigate generated signals and corresponding surveillance time series. It includes upload of csv or Excel files, data checks, possibility to set parameters for signal detection, visualisation of results and generation of reports. The tool is interactive and flexible so that it can be used in different contexts and customised to the user’s needs.
 
 ## Installation
-
 You can install the development version of SignalDetectionTool from [GitHub](https://github.com/United4Surveillance/signal-detection-tool) with:
```

Comment:
It says you can install the development version of the tool directly using devtools::install_github(), but isn't it actually the release version being installed this way, since it points to the default branch (which is main) - or has that been changed?

Reply:
Yes true, thanks! I also spotted that at some point but forgot again about it. This is still old when we did not have the develop branch yet.

Reply:
@PvomFelde Can you change the sentence to: You can install the most recent release of the SignalDetectionTool from